packages/wallets/src/features/cfd/modals/JurisdictionModal/JurisdictionModal.tsx

```diff
@@ -11,6 +11,7 @@ const JurisdictionModal = () => {
     const { isLoading } = useAvailableMT5Accounts();
 
     const marketType = modalState?.marketType || 'all';
+    const platform = modalState?.platform || 'mt5';
```

Comment:
why do we need modalState if we can directly pass prop to a modal ?

Reply:
Right now, we're using modalState instead of passing props, this is because it will be integrated into the FlowProvider in the future. So, for each modal and screen, we're trying to avoid passing props. This way, when we switch between different screens or modals, even if it's 2-3 steps ahead or behind, the state won't rely on the previous step's props.